packages/fxa-settings/src/pages/ResetPassword/index.tsx

```diff
@@ -0,0 +1,223 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Localized, useLocalization } from '@fluent/react';
+import React, { useCallback, useState } from 'react';
+import { useForm } from 'react-hook-form';
+import { useNavigate } from '@reach/router';
+import { HomePath } from '../../constants';
+import {
+  logViewEvent,
+  settingsViewName,
+  usePageViewEvent,
+} from '../../lib/metrics';
+import { useAccount, useAlertBar } from '../../models';
+import { InputText } from '../../components/Settings/InputText';
+import RememberPassword from '../../components/App/RememberPassword';
+
+// --forceEmail-- is a hint to the signup page that the user should not
+// be given the option to change their address
+// (e.g., if coming from a relying party)
+// forceEmail: account.get('email'),
+// forceEmail: this.relier.get('email')
+
+// --canGoBack-- determines if the user can navigate back to an fxa entrypoint
+
+// --serviceName-- is the relying party
+
+// TODO verify if these are passed as props or obtained a different way
+type ResetPasswordProps = {
+  forceEmail?: string;
+  canGoBack?: boolean;
+  serviceName?: string;
+};
+
+type FormData = {
+  email: string;
+};
+
+// eslint-disable-next-line no-empty-pattern
+const ResetPassword = ({
+  forceEmail,
+  canGoBack,
+  serviceName,
+}: ResetPasswordProps) => {
+  usePageViewEvent('settings.reset-password');
+
+  const displayedServiceName = serviceName || 'Account Settings';
+  const [email, setEmail] = useState<string>(forceEmail || '');
+  const [emailErrorText, setEmailErrorText] = useState<string>();
+  const [isFocused, setIsFocused] = useState(false);
+
+  const { handleSubmit } = useForm<FormData>({
+    mode: 'onBlur',
+    criteriaMode: 'all',
+    defaultValues: {
+      email: '',
+    },
+  });
+
+  const alertBar = useAlertBar();
+  const account = useAccount();
+  const { l10n } = useLocalization();
+  const navigate = useNavigate();
+
+  const onFocusMetricsEvent = 'reset-password.engage';
+
+  const onFocus = () => {
+    if (!isFocused && onFocusMetricsEvent) {
+      logViewEvent(settingsViewName, onFocusMetricsEvent);
+      setIsFocused(true);
+    }
+  };
+
+  const navigateToConfirmPwReset = useCallback(() => {
+    // TODO: passwordForgotToken
```

Comment:
What were you planning on doing with `clearQueryParams` and `passwordForgotToken`. I don't remember exactly why they would be needed here or how they are used. Maybe this can be taken out.

Reply:
I've added them here because they were mentioned in the original content-server view, but I haven't sussed out what they are used for yet (or if they're still needed). Marked as todo to investigate more.